Client/src/Pages/Authentication/login.js

```diff
@@ -33,139 +53,149 @@ function SignIn({t, i18n}) {
-          headers: {
-            Accept: "application/json",
-            "Content-Type": "application/json"
-          },
-        }
-      )
-      .then(response => {
-        return response.json()
+      "https://7jdf878rej.execute-api.us-east-2.amazonaws.com/test/users/" +
+        sessionStorage.getItem("id").toString() +
+        "/picture",
+      {
+        method: "GET",
+        headers: {
+          Accept: "application/json",
+          "Content-Type": "application/json",
+        },
+      }
+    )
+      .then((response) => {
+        return response.json();
       })
-      .then(json => {
-        console.log(json.picture)
+      .then((json) => {
+        console.log(json.picture);
         sessionStorage.setItem("image", json.picture);
         window.location.reload();
       })
-      .catch(error => {
+      .catch((error) => {
         console.log(error);
       });
   }
 
-
   //Gets run when submit is pressed and handles authentication.
-  const handleSubmit = async event =>{
+  const handleSubmit = async (event) => {
     event.preventDefault();
     var username = email;
-    try{
+    try {
       const user = await Auth.signIn(email, password);
       console.log(user);
       if (user.signInUserSession.accessToken !== undefined) {
-
         var authToken = user.signInUserSession.idToken.jwtToken;
-        var base64Url = authToken.split('.')[1];
+        var base64Url = authToken.split(".")[1];
         var json = JSON.parse(window.atob(base64Url));
         const cookies = new Cookies();
-        sessionStorage.setItem("token", user.signInUserSession.accessToken, { path: "/" });
+        sessionStorage.setItem("token", user.signInUserSession.accessToken, {
+          path: "/",
+        });
         sessionStorage.setItem("email", json.email);
         sessionStorage.setItem("firstName", json.given_name);
         sessionStorage.setItem("lastName", json.family_name);
         sessionStorage.setItem("role", "Student");
         sessionStorage.setItem("id", json.preferred_username);
         cookies.set("role", "Student", { path: "/" });
-        fetchPicture()
-      }
-    }catch(error){
-      if(error.code =="NotAuthorizedException"){
-        setValidInfo(false);   
+        fetchPicture();
       }
-      else{
-        alert(error.message)
+    } catch (error) {
+      if (error.code == "NotAuthorizedException") {
+        setValidInfo(false);
+      } else {
+        alert(error.message);
       }
       console.log(error);
     }
-  }
+  };
 
   //checks if they put in an email and password
   function validateForm() {
     return email.length > 0 && password.length > 0;
   }
 
-  //So the user can press enter rather than click the button
-  function handleKeyPress(event){
-    if(event.key === 'Enter' && validateForm()){
-      return handleSubmit
-    }
-  }
+  //hides and shows password
+  const togglePasswordVisiblity = () => {
+    setPasswordVisibility(isPasswordVisible ? false : true);
```

Comment:
You could use just `!isPasswordVisible` and avoid the ternary operator.

Reply:
Changed it. Thanks.